contentcuration/contentcuration/viewsets/contentnode.py

```diff
@@ -195,6 +196,7 @@ class ExtraFieldsSerializer(JSONFieldDictSerializer):
     mastery_model = ChoiceField(
         choices=exercises.MASTERY_MODELS, allow_null=True, required=False
     )
+    randomize = BooleanField()
```

Comment:
Could you either add this to this test here: https://github.com/learningequality/studio/blob/hotfixes/contentcuration/contentcuration/tests/viewsets/test_contentnode.py#L534 or create a similar test specifically for `randomize`?

Reply:
I added a section for `randomize` in th existing unit test